packages/react/src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { PropsWithChildren, ReactElement, ReactNode } from "react";
+import { OverlayArrow } from "react-aria-components";
+import {
+  mergeProps,
+  useOverlayPosition,
+  useTooltip,
+  useTooltipTrigger,
+} from "react-aria";
+import { useTooltipTriggerState } from "react-stately";
+import classNames from "classnames";
+
+const ANIMATION_DURATION = 200;
+
+export const Tooltip = ({
```

Comment:
You forgot to export it here I think: https://github.com/openfun/cunningham/blob/ddaf6aa33b007c940a026da98d19c0dd085c4c17/packages/react/src/index.ts#L29-L30

Reply:
Nice catch!